spec/lib/soapy_cake/integration_spec.rb

```diff
@@ -38,6 +38,17 @@
     )
   end
 
+  it 'does not parse a transaction_id as an integer', :vcr do
+    result = SoapyCake::Admin.new.conversions(
+      start_date: Date.new(2015, 4, 11),
+      end_date: Date.new(2015, 4, 12),
+      row_limit: 1
+    )
+
+    expect(result.count).to eq(1)
+    expect(result.first[:transaction_id]).to be_a(String)
```

Comment:
Let's check for the actual value to make sure `"123abc"` is not converted to `"123"`.